src/components/Section/Citizenship/Citizenship.jsx

```diff
@@ -1,198 +1,98 @@
 import { i18n } from '@config'
-import SectionElement from '@components/Section/SectionElement'
-import { SectionViews, SectionView } from '@components/Section/SectionView'
-import { Field } from '@components/Form'
-import Status from '@components/Section/Citizenship/Status'
-import Multiple from '@components/Section/Citizenship/Multiple'
-import Passports from '@components/Section/Citizenship/Multiple/Passports'
+import { ErrorList } from '@components/ErrorList'
+import SectionNavigation from '@components/Section/shared/SectionNavigation'
+import { CITIZENSHIP } from '@constants/sections'
+import * as formTypes from '@config/formTypes'
+import Intro from './Intro'
+import Status from './Status'
+import Multiple from './Multiple'
+import Passports from './Multiple/Passports'
+import Review from './Review'
 
-class Citizenship extends SectionElement {
-  render() {
-    return (
-      <div>
-        <SectionViews
-          current={this.props.subsection}
-          dispatch={this.props.dispatch}
-          update={this.props.update}>
+class Citizenship extends React.Component {
+  constructor(props) {
+    super(props)
 
-          <SectionView
-            name="intro"
-            back="relationships/review"
-            backLabel={i18n.t('relationships.destination.review')}
-            next="citizenship/status"
-            nextLabel={i18n.t('citizenship.destination.status')}>
-            <div>
-              <h1 className="section-header">{i18n.t('citizenship.intro.title')}</h1>
-              <Field
-                optional={true}
-                className="no-margin-bottom">
-                {i18n.m('citizenship.intro.body')}
-              </Field>
-            </div>
-          </SectionView>
+    this.form = formTypes[props.formType]
+    this.section = this.form.find(section => (section.key === CITIZENSHIP))
+
+    // Each key in subsectionLibrary corresponds to the subsection
+    // name in the config file.
+    this.subsectionLibrary = {
+      intro: Intro,
+      status: Status,
+      multiple: Multiple,
+      passports: Passports,
+      review: Review
+    }
+  }
+  getCitizenshipSubsections = () => {
 
-          <SectionView
-            name="review"
-            title={i18n.t('review.title')}
-            para={i18n.m('review.para')}
-            showTop={true}
-            back="citizenship/passports"
-            backLabel={i18n.t('citizenship.destination.passports')}
-            next="military/intro"
-            nextLabel={i18n.t('military.destination.intro')}>
-            <Status
-              name="status"
-              {...this.props.Status}
-              section="citizenship"
-              subsection="status"
-              dispatch={this.props.dispatch}
-              onUpdate={this.handleUpdate.bind(this, 'Status')}
-              onError={this.handleError}
-              required={true}
-              scrollIntoView={false}
-            />
+    return this.section.subsections.map(subsection => (
```

Comment:
Because we don't want to expose routes that aren't applicable to a form/section, I generated the routes from the section config files. this will only expose the sections applicable to this form.